Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/FacepunchMultiplayerMenu/FacepunchSteamworksController.cs

```diff
@@ -0,0 +1,42 @@
+﻿#if FACEPUNCH_STEAMWORKS
+using UnityEngine;
+using Steamworks;
+
+/// <summary>
+/// Singleton class to initialize Facepunch.Steamworks client and shotdown on application quit
+/// </summary>
+public class FacepunchSteamworksController : MonoBehaviour
+{
+	public static FacepunchSteamworksController facepunchSteamworksController;
```

Comment:
I'd personally name the variable `Instance` or `instance`. That would be more inline with the singletons used in other places eg. NetworkManager